pyproject.toml

```
@@ -49,6 +49,8 @@ dependencies = [
     "elastic-transport==8.4.1",
     "urllib3==1.26.18",
     "docker==6.0.0",
+    # avoid specific requests version to fix bug in docker-py
+    "requests > 2.26.0 != 2.32.0",
```

It looks like we need a comma here:
```suggestion
    "requests > 2.26.0,!= 2.32.0",
```